docs/packages/st.html

```diff
@@ -53,10 +53,15 @@
         <a class='octothorpe' href='#section-2'>#</a>
       </div>
       <h2>Description</h2>
-<p>This script can be used to retrieve list of clusters from the external data
+<p>This script can be used to perform several operations with external data
+pipeline usually deployed on Stage environment.</p>
+<p>First operation retrieves list of clusters from the external data
 pipeline through the standard REST API. Organization ID (a.k.a. account number)
```

Comment:
(just in case)
same as https://github.com/RedHatInsights/insights-results-aggregator-utils/pull/296#pullrequestreview-797441254

Reply:
updated as well